src/LoginButton.test.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import {shallow, mount} from "enzyme";
+import LoginButton from "./LoginButton.js";
+
+it("renders LoginButton without crashing", () => {
+  mount(<LoginButton />);
+});
+
+it("renders correctly (snapshot testing)", () => {
+  const wrapper = mount(<LoginButton />);
+  expect(wrapper).toMatchSnapshot();
+
+  wrapper.unmount();
+});
+
+it("verifies image for the button has right dimensions", () => {
```

Comment:
/s/verifies image/verifies that the image/

Reply:
Updated test names.